setup.py

```diff
@@ -14,10 +13,12 @@
 from setuptools.command.build_py import build_py
 from setuptools.command.develop import develop
 
-P2PD_VERSION = "v0.3.6"
-P2PD_CHECKSUM = "627d0c3b475a29331fdfd1667e828f6d"
-LIBP2P_TAR_URL = f"https://github.com/learning-at-home/go-libp2p-daemon/archive/refs/tags/{P2PD_VERSION}.tar.gz"
-P2PD_BINARY_URL = f"https://github.com/learning-at-home/go-libp2p-daemon/releases/download/{P2PD_VERSION}/p2pd"
+P2PD_VERSION = "v0.3.0-hivemind.8"
+
+P2PD_SOURCE_URL = f"https://github.com/learning-at-home/go-libp2p-daemon/archive/refs/tags/{P2PD_VERSION}.tar.gz"
+P2PD_BINARY_URL = f"https://github.com/learning-at-home/go-libp2p-daemon/releases/download/{P2PD_VERSION}/"
+EXECUTABLES = ["p2pd", "p2p-keygen"]
```

Comment:
To me, it's not entirely clear why we need `p2p-keygen` right now, especially since it can't be used in an installed version of hivemind by itself (it is not exposed via PATH)

Reply:
In that case, perhaps we should support this in the current PR to avoid downloading 8MB binaries that cannot be used?

Reply:
Exposing via `PATH` is indeed the next step, however, just having the binary is already *much* better than nothing.

Note that without this change, one needs to install Go, compile `p2p-keygen` and send it to the machine with the training monitor - all that just for launching a monitor that doesn't change its address when restarted.